Assets/Scripts/Source/Audio/AudioOptions.cs

```diff
@@ -29,12 +30,12 @@ public bool HasEndHandler
         }
     }
 
-    public float FadeInTime { get; private set; }
-    public float FadeOutTime { get; private set; }
-    public float Volume { get; private set; }
-    public bool Looping { get; private set; }
+    public float FadeInTime;
+    public float FadeOutTime;
+    public float Volume;
+    public bool Looping;
     // Should the AudioSource cutover without fading out the AudioClip?
-    public bool HardTransitionIn { get; private set; }
+    public bool HardTransitionIn;
 
     private Action onEndHandler;
```

Comment:
Maybe worth leveraging `UnityEvent` so we can also set this via the inspector: https://docs.unity3d.com/Manual/UnityEvents.html